packages/field-plugin/src/createFieldPlugin/createFieldPlugin.ts

```diff
@@ -55,6 +55,9 @@ export const createFieldPlugin: CreateFieldPlugin = ({
   const { uid, host } = params
 
   // ToDo: In development we need to load localhost:3300
+  // const origin = 'http://localhost:7070'
+  // const origin = 'http://localhost:3300'
```

Comment:
I think you can remove it

Reply:
Hey @ademarCardoso 
I prefer to continue this comment to make it easier for future developers to build the project to run locally.